src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -387,7 +397,13 @@ public void visitToken(DetailAST ast) {
         if (rcurly != null) {
             final String violation = validate(details);
             if (!violation.isEmpty()) {
-                log(rcurly, violation, "}", rcurly.getColumnNo() + 1);
+                if(ast.getType() == TokenTypes.VARIABLE_DEF){
+                    final DetailAST semiColon = details.nextToken;
+                    log(semiColon, violation, ";", rcurly.getColumnNo() + 2);
+                }
+                else {
+                    log(rcurly, violation, "}", rcurly.getColumnNo() + 1);
+                }
```

Comment:
@nrmancuso @romani 

I had to modify the ``visitToken()`` method otherwise it would be impossible to log violation message for ``;``. If you see properly, in the master branch, rightcurly logs violation for ``}`` only. To remove this false positive, I had to modify ``visitToken()`` like that. 

I am solve some false positive within the code, will push the changes very soon.